server/src/vcap-constants.es6

```diff
@@ -22,27 +22,27 @@ vcapConstants.bucket = intakeS3.credentials.bucket;
 const middlelayerService = vcapServices['user-provided'].find(element => {
   return element.name === 'middlelayer-service';
 });
-vcapConstants.middleLayerBaseUrl = middlelayerService.credentials.MIDDLELAYER_BASE_URL;
-vcapConstants.middleLayerUsername = middlelayerService.credentials.MIDDLELAYER_USERNAME;
-vcapConstants.middleLayerPassword = middlelayerService.credentials.MIDDLELAYER_PASSWORD;
+vcapConstants.middleLayerBaseUrl = middlelayerService.credentials.middlelayer_base_url;
+vcapConstants.middleLayerUsername = middlelayerService.credentials.middlelayer_username;
+vcapConstants.middleLayerPassword = middlelayerService.credentials.middlelayer_password;
 
 // Intake
 const intakeService = vcapServices['user-provided'].find(element => {
   return element.name === 'intake-client-service';
 });
-vcapConstants.intakeClientBaseUrl = intakeService.credentials.INTAKE_CLIENT_BASE_URL;
-vcapConstants.intakeUsername = intakeService.credentials.INTAKE_USERNAME;
-vcapConstants.intakePassword = intakeService.credentials.INTAKE_PASSWORD;
+vcapConstants.intakeClientBaseUrl = intakeService.credentials.intake_client_base_url;
+vcapConstants.intakeUsername = intakeService.credentials.intake_username;
```

Comment:
As mentioned in the README comments, I don't believe we are still using these so we can probably remove.